### DISCUSSION TITLE
Does the Streamlit version have network functionality?

### DISCUSSION THREAD

**Discussion by Mofjo** [hlky/stable-diffusion-webui#1424] [Q&A]
I'm just switching from the Gradio UI over to the Streamlit UI, and it seems great, but I haven't seen any documentation for how to run the Streamlit version over a network? Being able to use the Gradio UI on my phone is one of my favorite features.

Am I just missing something?

Thanks!

**Comment by codedealer** [comment]
Depends on where your phone is. It can be accessed from within the local network just like Gradio but it can't self host out of the box to be available online (like Gradio with `Share=True`).

**Reply by Mofjo** [reply]
Ah, I see. I never figured out the local network access option for Gradio. If the computer running SD is on the same wifi network as my phone, do I just point my phone browser to http://localhost:8501 like I would on my PC? Because I get an ERR_CONNECTION_REFUSED message when I try that.

**Reply by hlky** [reply]
Use the local ip address of your computer.

On Windows 10:

![image](https://user-images.githubusercontent.com/106811348/194063457-31d3d1d8-400c-44e5-9b06-3add2439ef32.png)

![image](https://user-images.githubusercontent.com/106811348/194063549-ba783eac-868b-4b2b-9303-ad6bca136654.png)

![image](https://user-images.githubusercontent.com/106811348/194063608-9054670b-6594-4a85-bf15-ce03e8a4e177.png)

`IPv4 address`

or

Open powershell/cmd
![WxvXn5eZUd](https://user-images.githubusercontent.com/106811348/194064431-32b9d0e5-10a8-4538-bd54-14443d86d2fc.png)
or
`win + r` -> `cmd`
Then run
`ipconfig`

**Reply by Mofjo** [reply]
Ah, of course -- thanks so much for the help! 🙏
